packages/app/src/server/service/search-delegator/elasticsearch.ts

```diff
@@ -91,9 +91,14 @@ class ElasticsearchDelegator implements SearchDelegator<Data> {
 
   initClient() {
     const { host, httpAuth, indexName } = this.getConnectionInfo();
+    const httpAuthArr = httpAuth.split(':');
+    const username = httpAuthArr[0];
+    const password = httpAuthArr[1];
+
     this.client = new elasticsearch.Client({
-      host,
-      httpAuth,
+      node: host,
+      ssl: { rejectUnauthorized: false }, // TODO: set ssl from global env config
```

Comment:
Please make a new task to fix this TODO.

Reply:
@yuki-takei actually ssl.rejectUnauthorized config does exist on our last elasticsearch, I thought it was new parameter that needs to add environment configuration.

Reference:
https://github.com/elastic/elasticsearch-js/issues/1208

Reply:
last versions rejectUnauthorized default value is false, current default value is true, so we need to put rejectUnauthorized manually.